kernel_gateway/services/activity/handlers.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import tornado.web
+import json
+from . import activity
+from ...mixins import TokenAuthorizationMixin, CORSMixin
+
+class ActivityHandler(TokenAuthorizationMixin, CORSMixin, tornado.web.RequestHandler):
+
+    def get(self, **kwargs):
+        if 'kg_list_kernels' not in self.settings or self.settings['kg_list_kernels'] != True:
+            raise tornado.web.HTTPError(403, 'Forbidden')
```

Comment:
I was referencing [this](https://github.com/jupyter-incubator/kernel_gateway/blob/master/kernel_gateway/services/kernels/handlers.py#L35) line of code, which I believe is the  `/api/kernels` endpoint.